### ISSUE TITLE
Spinner

### ISSUE THREAD

**Issue by poupotte** [cozy/cozy-guidelines#10]
I suggest to use spinner used by cozy-files-mobile in all applications : 
https://github.com/poupotte/cozy-files-mobile/blob/master/www-src/app/assets/img/spinner.svg

**Comment by m4dz** [comment]
Great! This spinner should be integrated to the Ui elements stylesheet directly and be called by setting an aria attribute `[aria-busy=true]` to elements uppon which we want the spinner be displayed (in a `after` pseudo-element).

**Comment by frankrousseau** [comment]
Sounds great. @poupotte and @m4dz could you add it to the guidelines? 

@m4dz I think we should put your GUI guidelines in a folder of this repo. What do you think about it?

**Comment by m4dz** [comment]
Yep, I'll add it to the guidelines when I'll start to work on them. After talking about with @poupotte, we decided to merge the Cozy-Ui issues in the guidelines repo, and keep mine only to sandbox an Ui framework in the first time :smile:.

**Comment by nono** [comment]
It's something for cozy-ui, not guidelines
